packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -1,1391 +1,1388 @@
-<template>
```

Comment:
moving of the `<template>` below `<script>` makes this file really difficult to review. perhaps we can revert that change in this PR, and create a new PR to update the order across all `.vue` files ?